### PR TITLE
feat(decorator): add onContextMenu to decorator

### PR DESCRIPTION
**Pull request by paul-balchin-ibm**
### Proposed changes

- Added onContextMenu callback function if decorator is an `href`
- Added onContextMenu callback function if decorator is the default, non-interactive `span`

### Testing instructions

- Regardless of the type of Decorator rendered (`span`, `href`, or `button`), if an `onContextMenu` prop is defined, a callback should be triggered.

### CODE REVIEW

**Review by areddon — APPROVED** [review]

File: src/components/DataDecorator/Decorator/__tests__/Decorator-test.js [hunk 1]
```diff
@@ -3,6 +3,7 @@
  * @copyright IBM Security 2019 - 2021
  */
 
+import { fireEvent } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import { render } from '@testing-library/react';
```
Comment (SimonFinney) [inline src/components/DataDecorator/Decorator/__tests__/Decorator-test.js:8]:
```suggestion
```

File: src/components/DataDecorator/Decorator/__tests__/Decorator-test.js [hunk 2]
```diff
@@ -3,6 +3,7 @@
  * @copyright IBM Security 2019 - 2021
  */
 
+import { fireEvent } from '@testing-library/react';
```
Comment (SimonFinney) [inline src/components/DataDecorator/Decorator/__tests__/Decorator-test.js:6]:
```suggestion
import { fireEvent, render } from '@testing-library/react';
```

**Review by SimonFinney — APPROVED** [review]
LGTM — thanks! I've one suggestion for consolidating the unit test imports.

**Comment by paul-balchin-ibm** [comment]
> LGTM — thanks! I've one suggestion for consolidating the unit test imports.

Of course. Thanks for seeing that.

**Pull request merged**